tests/migrations/test_executor.py

```diff
@@ -759,6 +759,17 @@ def apply(self, project_state, schema_editor, collect_sql=False):
             False,
         )
 
+    @mock.patch.object(MigrationRecorder, 'has_table', return_value=False)
+    def test_migrate_skips_schema_creation(self, mock):
+        """
+        The django_migrations table is not created if there are no migrations
+        to record.
+        """
+        executor = MigrationExecutor(connection)
+        # 0 queries, since the query for has_table is being mocked.
+        with self.assertNumQueries(0):
+            executor.migrate(targets=None, plan=[])
```

Comment:
`targets` is an empty list when `MIGRATE` is `False`:
```suggestion
            executor.migrate([], plan=[])
```